templates/shared/_featured_jobs.html

```diff
@@ -1,18 +1,16 @@
 <section>
   <div class="u-fixed-width">
-    <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Featured Jobs</h2>
+    <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Featured Roles</h2>
   </div>
   <div class="p-featured row u-sv2">
+    <hr class="u-fixed-width">
     {% for job in featured_jobs %}
       {% if loop.index < 7 %}
-        <div class="p-featured__item col-small-4 col-medium-3 col-4">
-          <p class="p-featured__role">
-             <a class="p-featured__url" href="/careers/{{job.id}}">{{ job.title }}</a>
-          </p>
+        <div class="p-featured__item col-small-4 col-medium-3 col-4 u-no-padding--left">
+          <div class="p-card p-featured__role u-no-margin--bottom">
```

Comment:
this has a lot pf classes, if you want to keep p-featured__item maybe simply extending the highligted card  placeholder would make this cleaner? e.g. @extend %p-card--highlighted` in the css

Reply:
Not sure if I'm misunderstanding what you meant but it looks a bit strange when I extend it in the item (pictured). Tried in the role as well but it messes with the height 
![Screenshot from 2023-02-03 13-35-55](https://user-images.githubusercontent.com/17607612/216605390-ea06e588-a7e1-45f6-a7ec-6f28d3377ad3.png)